target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -293,7 +293,9 @@ netgear,r6120)
 	ucidef_set_led_wlan "wlan2g" "WiFi 2.4GHz" "$boardname:green:wlan2g" "phy0tpt"
 	;;
 netgear,r6220|\
+netgear,r6260|\
 netgear,r6350|\
+netgear,r6850|\
 netgear,wndr3700-v5)
 	ucidef_set_led_switch "wan" "wan" "$boardname:green:wan" "switch0" "0x10"
 	set_wifi_led "$boardname:green:wifi"
```

Comment:
New devices should have it's wifi led trigger defined in the DTS (given the driver supports it). This should be the case for mt76 supported chips.

Reply:
changed that part. is it okay now?

Reply:
See my other comment.